tests/unit/plugins/test_event_consumer.py

```diff
@@ -15,7 +15,10 @@
 # limitations under the License.
 
 import asyncio
+import concurrent.futures
```

Comment:
FYI I wrote tests over the past week, and I definitely lost context when I left & came back. So, if something looks weird/off, it's probably me just being an idiot.